v2/interface.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2013-2017 The btcsuite developers
```

Comment:
looks like this can be removed or updated since it's a new package?

Reply:
removed